projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -26,9 +28,11 @@ public class ProjectCreationService {
   private static final Logger log = logger(ProjectCreationService.class);
 
   private final ProjectRepository projectRepository;
+  private final ProjectDataRepository projectDataRepository;
 
-  public ProjectCreationService(ProjectRepository projectRepository) {
+  public ProjectCreationService(ProjectRepository projectRepository, @Autowired ProjectDataRepository projectDataRepository) {
```

Comment:
Still open to discuss if we use autowired or not. Maybe @KochTobi or @sven1103 can state what is best practice here